### ISSUE TITLE
Difference Between 1 GPU training and Multiple GPU training

### ISSUE THREAD

**Issue by Adamdad** [CompVis/latent-diffusion#158]
Dear maintainers of latent-diffusion, a range of people and me found that, when training the latent diffusion model with `multiple gpu`, the generated image quality becomes disastrously bad. But when using a `single GPU`, the result seems to be much better. 

In your original paper, you are using a single node, but what is the difference in training with multiple GPUs? Actually, you did not put and batch operations in your model (no batch norm only group norm and layer norm). And the learning rate is also scaled by the batch size. I am not sure what is the main difference between those two.

And also, when training on large-sized datasets like LAION-5b, are you still using single GPU for training? If Yes, how long would it takes to train for one epoch?

Best!

**Comment by zhangdan8962** [comment]
Have you tried multiple GPU training?

**Comment by ader47** [comment]
Hi, @Adamdad, I have the same problem, have you figured this out?
